MIPs/mip-x-multichainapi.md

```diff
@@ -0,0 +1,236 @@
+---
+mip: x
+title: Adopt chain agnostic standards for a Multichain API
+status: Draft
+stability: n/a
+discussions-to: https://github.com/MetaMask/metamask-improvement-proposals/discussions/53
+author(s): Alex Donesky (@adonesky1), Jiexi Luan (@jiexi), Vandan Parikh(@vandan) 
+type: Maintainer
+created: 2024-10-28
+---
+
+## Summary
+This proposal recommends that MetaMask adopt a set of Chain Agnostic Improvement Proposals (CAIPs) to guide the design of a new Wallet API that is generalized for interactions between dapps and wallets in a multichain environment. A "Multichain API". Implementing these standards encourages broad adoption while enhancing interoperability, improving user experiences, and significantly streamlining development for multichain dapps.
+
+### Goals
+- Consistent Structure: Provide common patterns while separating namespaces of each ecosystem or network.
+- Interoperability: Employ standards to encourage the development of powerful libraries with support for multiple wallets and networks
+- Unlock the Design Space: Enable dapps to negotiate an interface with wallets for optimal multichain experiences.
+- Security: Enhance security and privacy associated with wallets delived through Browser Extensions.
```

Comment:
Kind of mentioned by the above points, but thoughts on `Extensibility` as it's own item? nbd

Reply:
Yeah, I think we should highlight the goal of improved Extensibility here. I'll adjust the bullet points 👍